community/modules/scheduler/schedmd-slurm-gcp-v6-controller/main.tf

```diff
@@ -111,17 +136,16 @@ module "slurm_cluster" {
   slurm_cluster_name = local.slurm_cluster_name
   region             = var.region
 
-  create_bucket = var.create_bucket
-  bucket_name   = var.bucket_name
+  create_bucket = false
+  bucket_name   = var.create_bucket ? module.bucket[0].name : var.bucket_name
   bucket_dir    = var.bucket_dir
 
   controller_instance_config = local.controller_instance_config
 
   enable_login = var.enable_login
   login_nodes  = var.login_nodes
 
-  nodeset     = var.nodeset
-  nodeset_tpu = var.nodeset_tpu
```

Comment:
This seems unrelated. Why is it deleted?

Reply:
hmm, strange rebase, reverted, thank you for spotting.